src/libs/AxelarQueryAPI.ts

```diff
@@ -96,18 +96,18 @@ export class AxelarQueryAPI {
   public async getEVMEvent(sourceChainId: string, srcTxHash: string, srcEventId: number) {
     await throwIfInvalidChainIds([sourceChainId], this.environment);
     await this.initQueryClientIfNeeded();
-    return this.axelarQueryClient.evm
-      .Event({
-        chain: sourceChainId,
-        eventId: `${srcTxHash}-${srcEventId}`,
-      })
-      .catch(() => undefined);
+    return this.axelarQueryClient.evm.Event({
+      chain: sourceChainId,
+      eventId: `${srcTxHash}-${srcEventId}`,
+    });
   }
 
   public async getConfirmationHeight(chain: string) {
     await throwIfInvalidChainIds([chain], this.environment);
     await this.initQueryClientIfNeeded();
-    return this.axelarQueryClient.evm.ConfirmationHeight({ chain });
+    const chains = await loadChains({ environment: this.environment });
```

Comment:
if we make this change, could you cache the result (the same way it does done in AxelarRecoveryAPI):

```
  protected async getChainInfo(chainId: string) {
    if (this.chainsList.length === 0) {
      this.chainsList = await loadChains({
        environment: this.environment,
      });
    }
```